base/files/scoped_file.cc

```diff
@@ -8,9 +8,9 @@
 #include "build/build_config.h"
 
 #if defined(STARBOARD)
+#include <unistd.h>
 #elif BUILDFLAG(IS_POSIX) || BUILDFLAG(IS_FUCHSIA)
 #include <errno.h>
-#include <unistd.h>
```

Comment:
This should be left here as well, e.g. original IS_POSIX code should be unchanged